candi/version_map.yml

```diff
@@ -188,7 +188,7 @@ k8s:
       kubeProxy: sha256:092f9526686d27964d17be772c42cde086690209cc8aea10c49991456eb879c2
   '1.20':
     status: available
-    patch: 15
```

Comment:
What's happening here and later on? Downgrade?

Reply:
I got these values from the old fork, fixed